docs/source/technical_tutorials/user_data/create-datasets-for-dashboard.ipynb

```diff
@@ -4,18 +4,22 @@
    "cell_type": "markdown",
```

Comment:
<p>"skip <s>to</s> Scenario 1 and go to Scenario 2"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/316/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>